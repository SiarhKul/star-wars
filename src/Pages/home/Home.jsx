import React from "react";
import logoStarWars from "../../assets/Star-Wars-Logo.png";
export const Home = () => {
	return (
		<div className="wrapper-logo">
			<img className="logo" src={logoStarWars} alt="log-sw" />
		</div>
	);
};
