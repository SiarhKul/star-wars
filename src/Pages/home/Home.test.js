import React from 'react';
import { Home } from './Home';

describe('Test <Home/>', () => {
  const component = shallow(<Home />);

  it('render snapshot <Home/> ', () => {
    expect(component).toMatchSnapshot();
  });

  it('should <Home/> has image', () => {
    expect(component.find('img')).toHaveLength(1);
  });
});
