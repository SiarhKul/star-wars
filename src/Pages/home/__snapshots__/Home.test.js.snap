// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Home/> render snapshot <Home/>  1`] = `
<div
  className="wrapper-logo"
>
  <img
    alt="log-sw"
    className="logo"
    src="test-file-stub"
  />
</div>
`;
